client/reader/sidebar/index.jsx

```diff
@@ -240,11 +243,17 @@ export class ReaderSidebar extends Component {
 					currentTag={ this.state.currentTag }
 				/>
 
-				<SidebarSeparator />
```

Comment:
`ReaderSidebarOrganizations` can render `null` so to avoid double separators using conditional.